TheExampleApp/Startup.cs

```diff
@@ -165,7 +167,8 @@ public class StagingMessageHandler : HttpClientHandler
         public string StagingHost { get; set; }
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
-            var req = request.RequestUri.ToString().Replace("contentful", StagingHost);
+            var regex = new Regex("contentful");
+            var req = regex.Replace(request.RequestUri.ToString(), StagingHost, 1);
```

Comment:
The `1` at the end means you only replace one occurrence, correct?

Reply:
yes